.github/workflows/buildtest.yaml

```diff
@@ -17,41 +17,43 @@ name: Test
 on:
   workflow_dispatch:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
-    - uses: actions/cache@v4
-      with:
-        path: |
-          ~/go/pkg/mod
-          ~/go/bin
-          ~/.cache
-        key: livekit-protocol
-
-    - name: Set up Go
-      uses: actions/setup-go@v5
-      with:
-        go-version: 1.21
-
-    - name: Set up gotestfmt
-      run: go install github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@v2.4.1
-
-    - name: Download Go modules
-      run: go mod download
-
-    - name: Static Check
-      uses: amarpal/staticcheck-action@master
-      with:
-        checks: '["all", "-ST1000", "-ST1003", "-ST1020", "-ST1021", "-ST1022", "-SA1019"]'
-        install-go: false
-
-    - name: Test
-      run: |
-        set -euo pipefail
-        MallocNanoZone=0 go test -race -json -v ./... 2>&1 | gotestfmt
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/go/pkg/mod
+            ~/go/bin
+            ~/.cache
+          key: livekit-protocol
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.21
+          cache-dependency-path: |
+            packages/golang/go.sum
```

Comment:
This is no longer necessary, right?

Reply:
ah, thanks! thought I caught all instances of that. fixed in [323b7c6](https://github.com/livekit/protocol/pull/629/commits/323b7c6b0969173afd0c60e972aaade14a6b9b4f)